app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageComposerInput.kt

```diff
@@ -244,22 +227,20 @@ private fun MessageComposerTextInput(
 }
 
 @Composable
-private fun CollapseButton(
+fun CollapseButton(
+    isCollapsed: Boolean,
     onCollapseClick: () -> Unit
 ) {
-    var isCollapsed by remember { mutableStateOf(false) }
-
     Box(
         contentAlignment = Alignment.Center,
         modifier = Modifier
             .fillMaxWidth()
             .wrapContentHeight()
     ) {
-        val collapseButtonRotationDegree by animateFloatAsState(targetValue = if (isCollapsed) 180f else 0f)
+        val collapseButtonRotationDegree by animateFloatAsState(targetValue = if (isCollapsed) 0F else 180f)
```

Comment:
question: why did it have to be inverted? Was it wrong before? 🤔

Reply:
I change logic from top which controls it and I need to invert it to make it work properly